app/server/src/models/annotation/body.model.js

```diff
@@ -0,0 +1,48 @@
+const mongoose = require("mongoose");
+
+const bodySchema = new mongoose.Schema(
+  {
+    purpose: {
+      type: String,
+    },
+    type: {
+      type: String,
+    },
+    value: {
+      type: String,
+    },
+    creator: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: "User",
+      required: true,
+    },
+    created: {
+      type: String,
+    },
+  },
+  { timestamps: true }
+);
+
+const Body = mongoose.model("Body", bodySchema);
+
+const createBody = async (purpose, type, value) => {
+  var body = new Body({
+    purpose,
+    type,
+    value,
+  });
+  body.created = body.createdAt;
+  const res = await body.save();
+  return res;
```

Comment:
Where do we create `creator` field?

Reply:
Also I am not sure whether we should user createdAt to assign body.created or not. I think created will be given as input

Reply:
Added!